WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -115,14 +151,33 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
         ReaderTracker.shared.stop(.readerPost)
     }
 
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        coordinator.animate(alongsideTransition: { _ in
+            self.featuredImage.deviceDidRotate()
```

Comment:
When I rotate the device the scroll goes back to the top. In the previous implementation, this wouldn't change the offset.

I know there's probably a good reason for that but I think this is really bad. If I'm reading an article and I rotate my device by mistake I'd have to scroll it all over again.

Reply:
I made a separate task so we can address this.

Reply:
Will be fixed in another PR.